src/components/Modal/BottomDockedModal/Backdrop/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import {View} from 'react-native';
+import Animated, {FadeIn, FadeOut} from 'react-native-reanimated';
+import type {BackdropProps} from '@components/Modal/BottomDockedModal/types';
+import {PressableWithFeedback} from '@components/Pressable';
+import useThemeStyles from '@hooks/useThemeStyles';
+
+function Backdrop({style, hasBackdrop, customBackdrop, onBackdropPress, animationInTiming = 300, animationOutTiming = 300, animationInDelay = 100}: BackdropProps) {
+    const styles = useThemeStyles();
+
+    if (!hasBackdrop) {
```

Comment:
Should it be here or should it be decided somewhere higher that the component shouldn't be added to hierarchy if there is no backdrop?